riscv/src/register/macros.rs

```diff
@@ -1,3 +1,9 @@
+/// `RV64`: Convenience macro to wrap the `csrrs` assembly instruction for reading a CSR register.
+///
+/// This macro should generally not be called directly.
```

Comment:
Not sure if this is actually correct, as some of the existing `register` definitions directly call this macro.

Open to wording suggestions about correct usage here, and other similar locations.